test/units/modules/cloud/openstack/test_os_server.py

```diff
@@ -84,6 +84,9 @@ def get_network(self, name):
     def get_openstack_vars(self, server):
         return server
 
+    def get_server(self, name):
```

Comment:
I don't see where this is called?

Reply:
The FakeCloud unit test needs to have a get_server attribute as it gets called from _present_server:
```
_________________ TestCreateServer.test_create_server_bad_nic __________________
[gw0] linux2 -- Python 2.7.15 /tmp/python-ftq_D0-ansible/python
self = <units.modules.cloud.openstack.test_os_server.TestCreateServer object at 0x7f312fc541d0>

    def test_create_server_bad_nic(self):
        '''
        - os_server:
            image: cirros
            flavor: m1.tiny
            nics:
              - net-name: missing_network
        '''
        with pytest.raises(AnsibleFail):
>           os_server._present_server(self.module, self.cloud)

test/units/modules/cloud/openstack/test_os_server.py:222: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

module = <MagicMock id='139849231581904'>
cloud = <units.modules.cloud.openstack.test_os_server.FakeCloud object at 0x7f312fc54290>

    def _present_server(module, cloud):
        changed = False
        diff = {'before': '', 'after': ''}
>       server = cloud.get_server(module.params['name'])
E       AttributeError: 'FakeCloud' object has no attribute 'get_server'
```